### ISSUE TITLE
Quibim precission tool cannot be launched

### ISSUE THREAD

**Issue by jpripen** [chaimeleon-eu/CHAIMELEON-Platform#9]
When I click on the Quibim Precision icon the screen shows the following:

![image](https://user-images.githubusercontent.com/60269099/229474107-b46a09e5-9444-4582-a279-85d8b4b2eade.png)

And nothing else.

**Comment by asalic** [comment]
Same here, getting the same HTML after I login.

**Comment by Palollo** [comment]
This is the same issue as #7, the response have changed but the cause is the same: normal users (not admin) don't have the role to access to Quibim Precision, not yet. Because the Quibim people still have to adjust the permissions/roles.

**Comment by kseymour** [comment]
I have the same problem. Any news on the correction of this issue? In the mean time and as a workaround, could you give admin rights to users?
Surprisingly, Thomas Trouillard is able to connect to the application. Is he going through a different authentification process?
He does not have access to the data though. He has requested admin rights some weeks ago to be able to view the data. We, at Medexprim, would like to access the datasets to participate to the curation effort and ensure that the data are properly imported after being exported from Medexprim Suite. Thank you!

**Comment by Palollo** [comment]
Yes, the permissions have been adjusted by Quibim people last week, so most users can access already to the Case Explorer. 
But there is still an issue with the users which has some special characters (like a dot ".") in the first part of their e-mail (the part before the "@"). I suppose they will notify when fixed.
Those are the cases of @jpripen and @sophiasw. 
But I think @kseymour already can access.
